src/redis.c

```diff
@@ -140,11 +143,14 @@ static redis_query_t *redis_config_query(oconfig_item_t *ci) /* {{{ */
   for (int i = 0; i < ci->children_num; i++) {
     oconfig_item_t *option = ci->children + i;
 
+    rq->db = REDIS_DEF_DB;
```

Comment:
This is wrong. The value will be overwritten when next option after `Database` will be proceeded.

Also, `calloc` initializes all bytes to 0. Do we really need REDIS_DEF_DB?
If you think yes, then just put this initialization above the loop.